doc/src/modules/rewriting.rst

```diff
@@ -36,21 +36,21 @@ them into a normal form. For this ``complex`` keyword is used. Note that it will
 always perform arithmetic expand to obtain the desired normal form:
 
     >>> (x + I*y).expand(complex=True)
-    re(x) + I*re(y) + I*im(x) - im(y)
+    re(x) - im(y) + I*re(y) + I*im(x)
 
     >>> sin(x + I*y).expand(complex=True)
-    sin(re(x) - im(y))*cosh(re(y) + im(x)) + I*cos(re(x) - im(y))*sinh(re(y) + im(x))
+    sin(re(x) - im(y))*cosh(im(x) + re(y)) + I*cos(re(x) - im(y))*sinh(im(x) + re(y))
 
 Note also that the same behavior can be obtained by using ``as_real_imag()``
 method. However it will return a tuple containing the real part in the first
 place and the imaginary part in the other. This can be also done in a two step
 process by using ``collect`` function:
 
     >>> (x + I*y).as_real_imag()
-    (re(x) - im(y), re(y) + im(x))
+    (re(x) - im(y), im(x) + re(y))
```

Comment:
I don't have a problem with this change but why has this changed? Evaluation should not affect this case.

Reply:
A different sorting method is being used and I'm trying to figure this out.

Reply:
it is now as it was originally -- no change there